app/src/androidTest/java/de/westnordost/streetcomplete/quests/AssertUtil.java

```diff
@@ -0,0 +1,44 @@
+package de.westnordost.streetcomplete.quests;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import de.westnordost.osmapi.map.data.Element;
+import de.westnordost.streetcomplete.data.osm.ElementGeometry;
+import de.westnordost.streetcomplete.data.osm.OsmElementQuestType;
+import de.westnordost.streetcomplete.data.osm.download.MapDataWithGeometryHandler;
+
+
+import de.westnordost.osmapi.map.data.BoundingBox;
+
+import static junit.framework.Assert.fail;
+
+
+public class AssertUtil {
+	public static void verifyYieldsNoQuest(OsmElementQuestType quest, BoundingBox bbox) {
+		MapDataWithGeometryHandler verifier = (element, geometry) ->
+		{
+			fail("Expected zero elements. Element returned: " +
+				element.getType().name() + "#" + element.getId());
+		};
+		quest.download(bbox, verifier);
+	}
+
+	class ElementCounter implements  MapDataWithGeometryHandler{
+		int count = 0;
+		@Override
+		public void handle(@NonNull Element element, @Nullable ElementGeometry geometry) {
+			Log.wtf("ElementCounter", element.getType().name() + "#" + element.getId());
+			count += 1;
+		}
+	}
+
+	public void verifyYieldsQuest(OsmElementQuestType quest, BoundingBox bbox) {
+		ElementCounter counter = new ElementCounter();
+		quest.download(bbox, counter);
+		if(counter.count == 0) {
+			fail("Expected nonzero elements. Elements not returned");
+		}
+	}
+}
```

Comment:
> I'd perhaps remove the wtf-log

Ops, that is debug code that I deleted. Maybe I failed to push the changes?

Reply:
I rebased and force-pushed what solved the issue.

Reply:
> Not sure why you are unhappy about it.

I though that it can be simplified and I am missing it, but if there are no obvious improvements then it improves my opinion about myself :)